### PR TITLE
Add guide for user research

### PR DESCRIPTION
**Pull request by johnwaterworth**
First version of the guide with introduction and guidance on getting informed consent.

### CODE REVIEW

**Comment by lozette** [comment]
The linter is failing on this PR because headers have to be surrounded by blank lines. Let me know if you need help resolving this!

**Comment by johnwaterworth** [comment]
Added blank lines, and fixed numbering in the ordered list.

File: guides/how-we-do-user-research.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# How we do user research
+
+## Introduction
+
+This guide explains how we do user research at dxw digital.
+
+The guidance builds on our user research [principles]() and [workflow](). It also builds on the [guidance for user research in the GOV.UK Service Manual](https://www.gov.uk/service-manual/user-research) so we can help teams meet the [Government Service Standard](https://www.gov.uk/service-manual/service-standard).
```
[the remark this answers is not in the document]
Comment (johnwaterworth) [reply guides/how-we-do-user-research.md]: I've added the urls. Was waiting for another pull request to get merged.

File: guides/how-we-do-user-research.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+- attach the consent form to an email in an accessible digital format
+- send a paper consent form in the post
+
+Ask the participant to send a reply, confirming their consent. They can reply to the email, or post back a signed paper consent form.
+
+#### During the session
+
+At the start of the session, remind the participant that their participation is voluntary and they can stop or take a break at any time. Remind the participant about the different ways you are recording the session, and about anyone observing or listening in. Ask the participant to confirm their consent for the recording.
+
+As the session continues, actively make sure you have the participants consent. For example, check in with the participant if they look concerned about an activity, or if you’re on a visit and  want to take a photo in a new area.
```
[the remark this answers is not in the document]
Comment (johnwaterworth) [reply guides/how-we-do-user-research.md]: Added the apostrophe. Thanks for spotting.

File: guides/how-we-do-user-research.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+There are two basic options:
+
+1. The simplest and best option is to use our client’s existing consent materials. If they don’t have any, we can work with the client to create consent materials for the project, that the client can then reuse in future.
+
+   In this case the client will manage all the research data we collect and keep the associated records of consent.
+
+1. If using the client’s consent materials is not possible, or if the client asks us to do confidential research, we can use the dxw digital [information sheet](https://drive.google.com/open?id=19lm5_gzzqzVP0NnNZZiqPhEwW2_i2bkI5apAaie4-4M) and [consent form](https://drive.google.com/open?id=1ilYXwrrkl9s4jMRkJm3uCWAON6bnEeYinKvPd-VOz8k) templates.
+
+   In this case the dxw will manage the research data and keep the associated records of consent.
```
[the remark this answers is not in the document]
Comment (johnwaterworth) [reply guides/how-we-do-user-research.md]: Removed the the.

**Pull request merged**